util/cpplint.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
Please remove this commit.  It is completely unrelated to making riscv work on DMTCP.

If you think that the developers of DMTCP should all agree to a newer version of cpplint.py, then the right way to do that, is to start a new PR, and let us comment on it.  Mixing that conversation with 'riscv' is confusing.

Also, another small point.  You're changint 'python3' to 'python'.  That breaks on some platforms, where 'python3' is the default.  It has been many years since people were using Python-2.x (which was what 'python' referred to, when python3 was introduced).